test/src/pages/details/instructor-resource-tab.test.js

```diff
@@ -14,42 +12,41 @@ const instructor = {
 };
 const pageData = transformData(details);
 
-const {instance: pane, ready} = instanceReady(InstructorResourceTab,
-    {
-        bookAbbreviation: 'BIOLOGY',
-        userStatusPromise: Promise.resolve(instructor),
-        model: {
-            resources: pageData.book_faculty_resources,
-            allies: pageData.book_allies,
-            freeStuff: {
-                heading: pageData.free_stuff_instructor.content.heading,
-                blurb: pageData.free_stuff_instructor.content.content,
-                loggedInBlurb: pageData.free_stuff_instructor.content.content_logged_in
-            },
-            webinar: {
-                text: pageData.webinar_content.content.heading,
-                url: pageData.webinar_content.link,
-                blurb: pageData.webinar_content.content.content
+describe('Instructor Resources Tab', () => {
+    it('matches snapshot', () => {
+        const wrapper = makeMountRender(InstructorResourceTabJsx, {
+            bookAbbreviation: 'BIOLOGY',
+            userStatusPromise: Promise.resolve(instructor),
+            model: {
+                resources: pageData.book_faculty_resources,
+                allies: pageData.book_allies,
+                freeStuff: {
+                    heading: pageData.free_stuff_instructor.content.heading,
+                    blurb: pageData.free_stuff_instructor.content.content,
+                    loggedInBlurb: pageData.free_stuff_instructor.content.content_logged_in
+                },
+                webinar: {
+                    text: pageData.webinar_content.content.heading,
+                    url: pageData.webinar_content.link,
+                    blurb: pageData.webinar_content.content.content
+                },
+                featuredResources: [],
+                otherResources: [],
+                communityResource: {
+                    heading: pageData.community_resource_heading,
+                    logoUrl: pageData.community_resource_logo_url,
+                    url: pageData.community_resource_url,
+                    cta: pageData.community_resource_cta,
+                    blurb: pageData.community_resource_blurb,
+                    featureUrl: pageData.community_resource_feature_link_url,
+                    featureText: pageData.community_resource_feature_text
+                }
             },
```

Comment:
tutor uses a lib I wrote called ObjectFactoryBot to make test data.  If you find yourself writing a lot of stuff like this you might check it out.  https://github.com/openstax/tutor-js/tree/master/tutor/specs/factories